blocks/editable/style.scss

```diff
@@ -3,6 +3,7 @@
 }
 
 .blocks-editable__tinymce {
+
```

Comment:
I agree that it does look out of place in places, it's been a tough one to solve

I've a [specific set of issues](https://github.com/WordPress-Coding-Standards/stylelint-config-wordpress/milestone/2) relating to _new lines_ that I need to run through, hopefully I can get to this in the next few days, I'll then refresh this PR an we can can go from there